pkg/vulnsrc/python/python.go

```diff
@@ -136,3 +137,24 @@ func (vs VulnSrc) Get(pkgName string) ([]Advisory, error) {
 	}
 	return results, nil
 }
+
+func decodeAdvisoryDB(r io.Reader) (AdvisoryDB, error) {
+	var obj map[string]interface{}
```

Comment:
```suggestion
	var obj map[string]json.RawMessage
```

Also, we may be able to use `json.RawMessage` in order to avoid marshaling and unmarshaling.
https://play.golang.org/p/4VFqyBdH9Iu